editor/example/spec/islandviewer.ts

```diff
@@ -0,0 +1,276 @@
+import type { GoslingSpec } from '@gosling.schema';
+
+const circularRadius = 250;
+const centerRadius = 0.5;
+
+const linearHeight = 120;
+const linearSize = linearHeight / 6;
+const spec: GoslingSpec = {
+    title: 'IslandViewer 4 (Bertelli et al. 2017)',
+    subtitle: 'Salmonella enterica subsp. enterica serovar Typhi Ty2, complete genome.',
+    assembly: [['NC_004631.1', 4791961]],
+    spacing: 50,
+    views: [
+        {
+            layout: 'circular',
+            static: true,
+            alignment: 'overlay',
```

Comment:
To close the gap on the top of the circular visualization

```suggestion
            alignment: 'overlay',
            "spacing": 0.1,
```